### ISSUE TITLE
Window jumps to top

### ISSUE THREAD

**Issue by armandsdz** [deanoemcke/thegreatsuspender#194]
Hi!

Could it be that The Great Suspender causes this to happen?
When a site is opened that is never before opened (i.e. there are no cookies set) and you scroll down while page is still loading, it jumps back to the top of the page when page is fully loaded.
I checked this by clearing cookies and then opening some site.
It didn't jump with The Great Suspended disabled but did when it's enabled.
It's opening ok when visiting second time.

Thanks!

**Comment by deanoemcke** [comment]
argh! yes, there's a ridiculous bug that you've just helped me find. looks like a typo. i blame my cats jumping on my keyboard.
i've made a fix - i'll try to get a new release out soon.

**Comment by armandsdz** [comment]
Great!
Let me know here when update is up so I can enable it again!
Thanks!

**Comment by deanoemcke** [comment]
@armandsdz there is a new version out on the chrome webstore (v6.16) which addresses the scrolling issue. let me know how you get on.

**Comment by armandsdz** [comment]
Jumping bug seams to be resolved!

Thanks!

**Comment by darsain** [comment]
Also driving me crazy. I see that I have 6.14 installed, but the latest version in chrome store is 6.19.

Any ideas why my extension isn't updating? Even when triggered manually via Developer Mode? Are newer versions released only to certain % of people perhaps?

I don't want to try remove & add extension again as I have quite a few tabs suspended right now :)

**Comment by deanoemcke** [comment]
@darsain unfortunately i've had to turn off automatic updates due to the potentially disastrous effect it can have (losing all suspended tabs with no record). i have done this (as you suspected) by only releasing the new version to 0% of people.
i'm not sure why the developer mode 'update extensions now' button does not do an update, but i guess it is due to the 0% rule i put in place. 
it's a real shame but i think the only way to upgrade is to do an uninstall / reinstall. in doing this, you will lose any settings and also any saved sessions (within the extension session manager). also, make sure you unsuspend all tabs before doing this as they will disappear when you uninstall.
another idea, if unsuspending fills you with dread, is to install session buddy and save the current session. this will save suspended urls. when uninstalling / reinstalling you will still have the same extension id, so you should then be able to reload the session via session buddy and all your tabs will remain suspended.
this *should* work :D

**Comment by darsain** [comment]
Oh. That's annoying :)

**Comment by korzynski** [comment]
I'm running the latest version (6.21) but have been encountering this issue for weeks. Pages randomly scroll downward several lines after loading. I just stumbled on this thread while doing a google search, thinking it was a bug in Chrome.

**Comment by armandsdz** [comment]
Now when you say so, I too have observed such behavior...haven't done any research if this is related to Great Suspender..but it may as well be so...
I was thinking that it is a strange bug in websites I make but nobody else complained about it..so, I just forgot about it.
But haven't seen this for a while..

**Comment by MinimaxHeadroom** [comment]
:+1: the scrolling's happening for me as well. Running Chromium on Max OS X Yosemite.

**Comment by rickydeez** [comment]
> I'm running the latest version (6.21) but have been encountering this issue for weeks. Pages randomly scroll downward several lines after loading. I just stumbled on this thread while doing a google search, thinking it was a bug in Chrome.

Same here.... this isn't fixed.

**Comment by giolook** [comment]
@deanoemcke hi Dean, just like the authors of the last 4 posts here, I also have this issue from time to time. 

Also, when I click Play on an Adobe Flash player embedded in a page, it sometimes starts playback with volume down to zero. The volume in this player can be controlled by up/down keys on the keyboard. This effect does not manifest itself in players that don't have key bindings. It may also matter that the page with the player typically contains a lot of ads, in particular, sometimes - an ad popup window that's activated by the first click on the region with the player. However, I use AdBlock - it blocks these windows, sometimes faster than I can notice them. So maybe the issue appears on a page that's already loaded just because a popup window is activated on it by my click.

Could you please look into this issue again?

**Comment by amanjee** [comment]
I have this same issue, i narrowed it down to Great Suspender. As soon as I load any page (refresh, new tab, etc) It paginates down. It is a very frustrating bug. That said, if you need logs or something, feel free to ask me with some directions and I will try to provide any add'tl info to help trace this bug.

**Comment by 0reo** [comment]
I'm also still having this issue, v6.21, chrome 53, ubuntu 16.04

**Comment by kmactavish** [comment]
Can this issue be re-opened? Looks like people (including me) are still experiencing it.

v6.21, chrome 54, ubuntu 14.04

**Comment by kschat** [comment]
So I did some digging into why this is happening as I was planning on putting in a PR to fix this issue. It turns out the issue is due to how the position cookie is stored (per host instead of per tab ID). After discovering this I pulled down the repo, and started working on a fix...until I realized the version I have installed (latest on Chrome Web Store; 6.21) does not match the latest version in the [`manifest.json`](https://github.com/deanoemcke/thegreatsuspender/blob/86cbc344a676482988d59df915b7c45bbff8a605/src/manifest.json#L4), 6.30.

It appears 6.30 (or maybe a version between 6.30 and 6.21, I didn't check) fixes this issue by adding the tab ID to the cookie name. Is there a reason this hasn't been released yet?
